src/setup-dotnet.ts

```diff
@@ -13,26 +13,29 @@ export async function run() {
     // If a valid version still can't be identified, nothing will be installed.
     // Proxy, auth, (etc) are still set up, even if no version is identified
     //
-    let version = core.getInput('dotnet-version');
-    if (!version) {
+    let versions = core.getMultilineInput('dotnet-version');
+    if (!versions.length) {
       // Try to fall back to global.json
       core.debug('No version found, trying to find version from global.json');
       const globalJsonPath = path.join(process.cwd(), 'global.json');
       if (fs.existsSync(globalJsonPath)) {
-        version = getVersionFromGlobalJson(globalJsonPath);
+        versions[0] = getVersionFromGlobalJson(globalJsonPath);
```

Comment:
nitpick: `versions.push(getVersionFromGlobalJson(globalJsonPath))`

Reply:
Do you want to add this statement ? 
After version[0] ? 
Declaration?

Reply:
@hkaur008 this has been resolved